pkgs/tools/misc/broot/default.nix

```
@@ -27,6 +35,23 @@ rustPlatform.buildRustPackage rec {
   '';
 
   postInstall = ''
+    # Do not nag users about installing shell integration, since
+    # it is impure.
+    wrapProgram $out/bin/broot \
+      --set BR_INSTALL no
+
+    # Install shell function for bash.
+    $out/bin/broot --print-shell-function bash > br.bash
+    install -Dm0555 br.bash $out/etc/profile.d/br.sh
```

This needs to be sourced, not executed.

Also it looks like the function implementation uses bashisms, so we should retain the `.bash` suffix so as to not make anyone think it's POSIX-compliant sh (it uses the `function` keyword and `$(<file)`).

```suggestion
    install -Dm0444 -t $out/etc/profile.d br.bash
```

---

Sounds good, I have added this change.